hapi/hedera-protobufs/block/block_service.proto

```diff
@@ -353,6 +353,18 @@ enum PublishStreamResponseCode {
      * block stream data.
      */
     STREAM_ITEMS_BEHIND = 5;
+
+    /**
+     * The block node had an internal error and cannot continue processing.<br/>
+     * The publisher may retry again later.
+     */
+    STREAM_ITEMS_INTERNAL_ERROR = 6;
+
+    /**
+      * The requested stream is not available.<br/>
+      * The publisher may retry again later.
```

Comment:
Minor: `may` at 359 and 365 may or may not require capitalization for consistency with other docs here.

Reply:
It's not intended to be a keyword in those two cases, merely a bit of information.

Reply:
Sounds good.